src/commands/deployment/gate.ts

```diff
@@ -80,6 +83,11 @@ export class DeploymentGateCommand extends Command {
     description:
       'When true, the script will consider the gate as failed when timeout is reached or unexpected errors occur calling the Datadog APIs',
   })
+  // monitorVariable is hidden because it's not available yet
+  private monitorVariable = Option.String('--monitor-variable', '', {
+    validator: t.isString(),
+    hidden: true,
+  })
```

Comment:
I want to deliberately hide it from the command help because this is a trial we are running with a couple of customers and we are not sure if this will be the definitive approach or if this will change radically in the near future